clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -10804,12 +10802,20 @@ void LinkerWrapper::ConstructJob(Compilation &C, const JobAction &JA,
     for (auto &I :
          llvm::make_range(ToolChainRange.first, ToolChainRange.second)) {
       const ToolChain *TC = I.second;
-      if (TC->getTriple().isSPIR() &&
+      if ((TC->getTriple().isSPIR() || TC->getTriple().isSPIRV()) &&
```

Comment:
Please fix the comments above as well, to include spirv64.

Reply:
Done in upcoming commit. Thanks.